sfge/src/main/java/com/salesforce/rules/UnusedMethodRule.java

```diff
@@ -124,6 +124,8 @@ private boolean methodIsIneligible(MethodVertex vertex) {
                 || directedToSkip(vertex)
                 // Abstract methods are ineligible.
                 || vertex.isAbstract()
+                // Methods whose name starts with "__sfdc_" are getters/setters, and are ineligible.
```

Comment:
This is the new exclusion that we discussed yesterday.

Reply:
For future reference, could you add a comment explaining why we decided getters/setters don't need to be checked? Same with abstract check above. That way we have some credibility in our stance.